clients/src/main/java/org/apache/kafka/clients/admin/Admin.java

```diff
@@ -34,6 +27,14 @@
 import org.apache.kafka.common.annotation.InterfaceStability;
 import org.apache.kafka.common.config.ConfigResource;
 
+import java.time.Duration;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
```

Comment:
Avoid unnecessary reorderings (this class does not contain any actual code change).